src/Hashids.net/Hashids.cs

```diff
@@ -207,6 +207,92 @@ public string EncodeLong(long number)
         /// <returns>Array of 64-bit integers.</returns>
         public long[] DecodeLong(string hash) => GetNumbersFrom(hash);
 
+        /// <summary>
+        /// Decodes the provided hash into numbers.
```

Comment:
Why "numbers" is in plural form? And actually it will be better to use inheritdoc for implementation methods to avoid duplications and such small typos.

Reply:
I would say yes, but maybe @manigandham has other thoughts about such unrelated changes?